packages/tree-view-for-tests/src/index.ts

```diff
@@ -43,7 +42,7 @@ function tree(
       `${times('#', level)}${path.basename(_path)}`,
       options.hashIgnores.includes(path.basename(_path))
         ? undefined
-        : `#${hash(fs.readFileSync(_path, 'utf8'))}`,
+        : `#${hash(fs.readFileSync(_path, 'utf8').split('\r').join(''))}`,
```

Comment:
how about this - 
```suggestion
        : `#${hash(fs.readFileSync(_path, 'utf8').replace(/\r/gm, ''))}`,
```

(haven't tested, just curious)